airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/errorreporter/JobErrorReporter.java

```diff
@@ -47,6 +49,8 @@ public class JobErrorReporter {
   private static final String NORMALIZATION_REPOSITORY_META_KEY = "normalization_repository";
   private static final String JOB_ID_KEY = "job_id";
 
+  private static final ImmutableSet<FailureType> SUPPORTED_FAILURETYPES = ImmutableSet.of(FailureType.SYSTEM_ERROR);
+
```

Comment:
Not that big of a deal, but I think in this case an exclusion list might be more appropriate than inclusion list.  There's only one type of error we want to exclude - we want everything else (even those errors which are malformed and might be missing this type)

Reply:
I think an exclusion list makes sense. I do think we should leave out manual_cancellation as well. Let me know if there is something there that I am missing.